CorsixTH/Lua/entities/humanoids/grim_reaper.lua

```diff
@@ -38,3 +38,12 @@ end
 function GrimReaper:tickDay()
   return false
 end
+
+function GrimReaper:afterLoad(old, new)
+  if old < 213 and new >= 213 then
+    self.mood_marker = 2
+  end
+
+  self:updateDynamicInfo()
```

Comment:
Does the grim reaper even have dynamic info to update? Or the Inspector or VIP for that matter.